CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 ### Fixed
 - Select disabled background color
 
+### Refactored
```

Comment:
please rebase from master first then add this to the version after the latest. for example if latest version is 0.4.5, we can add this to 0.4.6 (Unreleased)